voctocore/lib/commands.py

```diff
@@ -60,11 +87,38 @@ def set_composite_mode(self, composite_mode):
 		self.pipeline.vmix.setCompositeMode(mode)
 		return True
 
-	def set_stream_blank(self, src_name_or_id):
-		src_id = self.decodeBlankerSourceName(src_name_or_id)
-		self.pipeline.streamblanker.setBlankSource(src_id)
-		return True
+	def get_composite_mode(self):
+		try:
+			mode = self.pipeline.vmix.getCompositeMode()
+			name = str(mode).split('.')[-1]
+			return (True, name)
+		except Exception as e:
+			raise KeyError("composite-mode %s unknown" % mode)
+
+	def set_stream_status(self, *args):
+		try:
+			if args[0] == "live":
+				self.pipeline.streamblanker.setBlankSource(None)
+				return True
+			elif args [0] == "blank":
+				src_id = self.decodeBlankerSourceName(args[1])
+				self.pipeline.streamblanker.setBlankSource(src_id)
+				return True
+			else:
+				return (False, "invalid arguments")
+		except KeyError as e:
+			raise KeyError("invalid arguments")
```

Comment:
where does this exception come from? is it from decodeBlankerSourceName? shouldn't it then be insise the elif? also, a ore descriptive explanation of the error would be nice